docs/src/pages/markdown-page.md

```
@@ -2,4 +2,6 @@
 title: Markdown page example
 ---
 
+# Markdown page example
```

Markdownlint (0.37.0)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)